decidim/decidim
# Fix invalid rendering of meeting and proposal body texts

#### :tophat: What? Why?
Currently meeting and proposal body rendering can be broken as explained at #9763. This fixes the described issue.

#### :pushpin: Related Issues
- Fixes #9763

#### Testing
- Create a meeting from admin panel
- Add a list to the meeting description from the WYSIWYG editor
- Go to see the meeting page
- Check the page source
- See that there are no extra `<br>` tags within the meeting description list
- Repeat the same process for admin created proposal
